release/src/main/scripts/script.config

```diff
@@ -0,0 +1,139 @@
+  RELEASE_VER
+  RC_NUM
+  REPO_URL
+  INSTALL_GCLOUD
+  USER_GCP_PROJECT
+  USER_GCS_BUCKET
+  USER_SERVICE_ACCOUNT_EMAIL
+  INSTALL_HUB
+  USER_REMOTE_URL
+  GITHUB_USERNAME
+  INSTALL_GNOME_TERMINAL
+  LOCAL_BEAM_DIR
+  java_quickstart_direct
+  java_quickstart_apex_local
+  java_quickstart_flink_local
+  java_quickstart_spark_local
+  java_quickstart_dataflow
+  java_mobile_game
+  python_quickstart_mobile_game
+  python_leaderboard_direct
+  python_leaderboard_dataflow
+  python_gamestats_direct
+  python_gamestats_dataflow
+)
+
+
+########################################
+# Environment Configurations
+########################################
+
+# Beam version of current release
+# e.g. 2.14.0
+RELEASE_VER=
+
+# Release candidate number
+# This is an identifier for each candidate we built for release. Start from 1
+# and increment if new candidate is built.
+# e.g. 1
+RC_NUM=
+
+# The repo URL from the vote email sent by Release Manager
+# e.g. https://repository.apache.org/content/repositories/orgapachebeam-0000
+REPO_URL=
+
+# Install Google Cloud SDK
+# Google Cloud SDK is required to run validation pipeline on DataflowRunner.
+# Set to true so that it will be installed if not found from local.
+INSTALL_GCLOUD=true
+
+# GCP project id
+# Required for running pipeline with DataflowRunner for validation.
+# e.g. apache-beam-testing
+USER_GCP_PROJECT=
+
+# GCS bucket name
+# Required for running pipeline with DataflowRunner for validation.
+# e.g. bucket-name (do not include 'gs://')
+USER_GCS_BUCKET=
+
+# GCP service account email
+# This service account should be under your project ${USER_GCP_PROJECT}. If you
+# don't have one, create it from GCP IAM console and set as project owner. You
+# can leave it empty if you have gcloud setup before.
+USER_SERVICE_ACCOUNT_EMAIL=
+
+# GCP application credentials
+# This is used to setup gcloud. Override following value if you already set
+# it in bashrc.
+GOOGLE_APPLICATION_CREDENTIALS=~/.config/gcloud/application_default_credentials.json
+
+# Install hub
+# hub is a tool used in run_rc_validation.sh and verify_release_build.sh to
+# create a Github PR in an automatic way. Set to true so that it will be
+# installed if not found from local.
+INSTALL_HUB=true
+
+# Your Beam repo URL
+# Used by hub to create a PR for validation. This is your personal Beam repo
+# but forked from apache/beam.
+# e.g. https://github.com/your-github-name/beam.git
+USER_REMOTE_URL=
+
+# Your github username
+# Used by hub to create a PR for validation.
+GITHUB_USERNAME=
+
+# Install gnome-terminal
+# Used in Python Leaderboard & GameStates to launches data injection pipeline
+# in a separate terminal. Set to true so that it will be installed if not found
+# from local. Otherwise, validation on Python Leaderboard & GameStates will be
+# skipped.
+INSTALL_GNOME_TERMINAL=true
+
+# Local Beam directory
+# This is a local workspace used by release scripts.
+LOCAL_BEAM_DIR="$(mktemp -d -t beam-release.XXXXXX)"
```

Comment:
why `XXXXXX`? does it make sense to add `RELEASE_VER` to it as well?

Reply:
`XXXXXX` is required and will be replaced by random characters. The whole string `beam-release.XXXXXX` is a [template](https://www.gnu.org/software/autogen/mktemp.html). 

Add `RELEASE_VER` sgtm.